README.md

```diff
@@ -15,6 +15,23 @@
 <br>
 <br>
 
+## Usage
+
+Add linters to your project :
```

Comment:
We might want to link to local READMEs ?
- https://github.com/SocialGouv/linters/tree/master/packages/eslint-config-react
- https://github.com/SocialGouv/linters/tree/master/packages/eslint-config-recommended
- https://github.com/SocialGouv/linters/tree/master/packages/tslint-config-recommended